packages/ts/react-crud/src/util.ts

```diff
@@ -12,8 +18,12 @@ export function convertToTitleCase(inputString: string): string {
   return titleCaseWords.join(' ');
 }
 
-export type ComponentStyleProps = Readonly<{
-  id?: string;
-  style?: CSSProperties;
-  className?: string;
-}>;
+export function registerStylesheet(stylesheet: CSSStyleSheet): void {
+  const css = Array.from(stylesheet.cssRules)
+    .map((rule) => rule.cssText)
+    .join('\n');
+
+  const styleTag = document.createElement('style');
+  styleTag.textContent = css;
+  document.head.insertAdjacentElement('afterbegin', styleTag);
```

Comment:
Would the `document.head.prepend` work here? It's shorter.

Reply:
Yep, that works as well. Changed.